doc/user/content/known-limitations.md

```diff
@@ -25,6 +25,10 @@ tag](https://github.com/MaterializeInc/materialize/issues?q=is%3Aopen+is%3Aissue
 
 - `ANY` and `ALL` queries generate suboptimal plans {{% gh 744 %}}
 
+### Functions
+
+- Materialize does not support [window functions](https://www.postgresql.org/docs/current/tutorial-window.html). In some cases, you may be able to achieve the desired results with [temporal filters](https://materialize.com/temporal-filters/) or the [TOP K by group](../sql/idioms#top-k-by-group) idiom instead. {{% gh 213 %}}
```

Comment:
Wdyt of adding "at this time" to make it (maybe too subtly) obvious that we would be interested in adding support for window functions (maybe, eventually).

Reply:
That's everything on this page, though! The top of the page calls that out explicitly, and invites users to engage on GitHub:

> The following links describe features that Materialize does not yet support, but plans to in future releases. If any of these issues impact you, feel free to let us know on the linked-to GitHub issue.

Reply:
Yeah, I left "at this time" or "currently" out because (1) notice at the top; (2) I'm wary of indicating it's a temporary status when we don't have a timeline for implementation (and don't seem 100% sure we want to implement it, from the #213 discussion).